Workbooks/UpdateCompliance/GetStartedWithUpdateCompliance/Get started with Update Compliance.workbook

```diff
@@ -0,0 +1,327 @@
+              "json": "**New policy required**\r\n\r\nA new policy is required to use Update Compliance: \"AllowUpdateComplianceProcessing\". If you're already using Update Compliance and have configured your devices prior to May 10, 2021, you must configure devices with this additional policy. \r\n<br></br>\r\n\r\n**Azure AD required**\r\n\r\nBeginning October 15, 2022, devices that are neither joined nor hybrid joined to Azure AD will no longer appear in Update Compliance. Learn more on how to enroll your devices into Azure AD <a href='' target='_blank'>How to: Plan your Azure AD join implementation</a>\r\n<br></br>\r\n\r\n**Log Analytics Workspace required**\r\n\r\nChoose an existing or configure a new Log Analytics Workspace. If you don't have one, see how to <a href='' target='_blank'>Create a Log Analytics workspace in the Azure portal</a>\r\n<br></br>"
+            },
+            "conditionalVisibility": {
+              "parameterName": "currentTab",
+              "comparison": "isEqualTo",
+              "value": "Prerequisites"
+            },
+            "name": "text - 1"
+          },
+          {
+            "type": 11,
+            "content": {
+              "version": "LinkItem/1.0",
+              "style": "paragraph",
+              "links": [
+                {
+                  "id": "8b5c667e-d1d4-4925-8600-334850500769",
+                  "cellValue": "currentTab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Previous",
+                  "subTarget": "Prerequisites",
+                  "style": "secondary",
+                  "linkIsContextBlade": true
+                },
+                {
+                  "id": "56386609-9d23-4920-89d5-155f0c5e7340",
+                  "cellValue": "currentTab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Next",
+                  "subTarget": "Configure",
+                  "style": "primary"
+                }
+              ]
+            },
+            "name": "footer nav 1"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "currentTab",
+        "comparison": "isEqualTo",
+        "value": "Prerequisites"
+      },
+      "name": "PrerequisiteGroup"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "loadType": "always",
+        "items": [
+          {
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "parameters": [
+                {
+                  "id": "e596b4fd-38c2-4e1a-bfeb-86c440e19c8a",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "Subscription",
+                  "label": "Azure Subscription",
+                  "type": 6,
+                  "description": "Select an Azure Subscription",
+                  "isRequired": true,
+                  "isGlobal": true,
```

Comment:
do these need to be global? if nothing downstream modifies these values, don't make them global